flake.nix

```diff
@@ -2,15 +2,17 @@
   description = "Ply - A helper library for working with compiled, parameterized Plutus Scripts";
 
   inputs = rec {
-    haskell-nix.url = "github:mlabs-haskell/haskell.nix";
-
+    haskell-nix.url = "github:input-output-hk/haskell.nix";
```

Comment:
@TotallyNotChase it might be this!

Reply:
we're supposed to use official haskell nix now - I think this should still pass fine (the previous commit with this passed)

Reply:
```
bladyjoker@morphism:~/Desktop/ply$ nix flake metadata github:input-output-hk/haskell.nix/e3397e0ace4aeadefe941241bd5b7bfefad850a9
Resolved URL:  github:input-output-hk/haskell.nix/e3397e0ace4aeadefe941241bd5b7bfefad850a9
Locked URL:    github:input-output-hk/haskell.nix/e3397e0ace4aeadefe941241bd5b7bfefad850a9
Description:   Alternative Haskell Infrastructure for Nixpkgs
Path:          /nix/store/k35qjhl7mfl0bnq6iaj22rhcpz3rbdki-source
Revision:      e3397e0ace4aeadefe941241bd5b7bfefad850a9
Last modified: 2022-07-26 23:10:13
```

Yeah it's likely rebuilding the world because of this but who knows :shrug:

Reply:
screw it, yolo